defaults/k8s_eks_config.yaml

```diff
@@ -16,7 +16,7 @@ eks_vpc_cni_version: 'v1.7.5-eksbuild.1'
 eks_nodegroup_role_arn: 'arn:aws:iam::797456418907:role/helm-test-worker-nodes-NodeInstanceRole-6ACHDYEKNN3I'
 
 scylla_version: '4.2.1'
-scylla_mgmt_agent_version: '2.2.0'
```

Comment:
What is the problem we have with the existing versions that we use it for "fix"?

Reply:
Moved to separate commit added info to the commit message